joplin/js/CreateContentModal/index.js

```diff
@@ -35,6 +35,7 @@ class CreateContentModal extends Component {
       creatingContent: false,
       content_or_topic: 'content',
       missingTitle: false,
+      can_set_topics: (document.getElementById("createNewContentButton").dataset.isEditor === 'false')
```

Comment:
Reversing the logic could make this more future proof if we need to add more roles. So instead of saying everyone except an editor can set topics, we can make it more explicit by saying only moderators and superadmins can set topics. And that logic to determine who can_set_topics can be done in the joplin_tags/py.

Using the builtin "json_script" tag lets you not have to hack your own variable injection, and it'll let you use JSON spec which allows Boolean values. It also can give us a consistent way to organize how, where, and which python values are getting sent to js. 

https://docs.djangoproject.com/en/3.0/ref/templates/builtins/#json-script
ex:
`{{ style_guide_url|json_script:"style-guide-url" }}` in template
`JSON.parse(document.getElementById('style-guide-url').textContent)` in js

Reply:
incorporated the feedback!